### PR TITLE
Add coq-of-ocaml.2.5.2

### PR DESCRIPTION
**Pull request by clarus**
Add `coq-of-ocaml.2.5.2` with support for both OCaml 4.12 and OCaml 4.13.

### CODE REVIEW

File: packages/coq-of-ocaml/coq-of-ocaml.2.5.2+4.12/opam [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  ["sh" "-c" "cd proofs && ./configure.sh"] {coq:installed}
+  [make "-C" "proofs" "-j%{jobs}%"] {coq:installed}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+install: [
+  [make "-C" "proofs" "install"] {coq:installed}
+]
+depends: [
+  "csexp"
+  "dune" {>= "2.8"}
```
Comment (kit-ty-kate) [inline packages/coq-of-ocaml/coq-of-ocaml.2.5.2+4.12/opam:18]:
```suggestion
  "dune" {>= "2.9"}
```

File: packages/coq-of-ocaml/coq-of-ocaml.2.5.2+4.12/opam [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  "ocaml" {>= "4.12" & < "4.13"}
```
Comment (kit-ty-kate) [inline packages/coq-of-ocaml/coq-of-ocaml.2.5.2+4.12/opam:19]:
```suggestion
  "ocaml" {>= "4.12" & < "4.14"}
```
Per your own comment

**Comment by kit-ty-kate** [comment]
(oops sorry disregard my two previous comments, I didn’t notice there was two releases in this one PR)

I’ve fixed the constraints in coq-of-ocaml.2.5.2+4.13 though. Could you return it in the opam file upstream so that it can be taken into account for next time?

**Comment by clarus** [comment]
OK, thanks! I did the change upstream.

**Pull request merged**